src/components/Retool.js

```diff
@@ -1,9 +1,65 @@
 import { useEffect, useRef, useState } from "react";
 
-const Retool = ({ data, url, height, width, onData }) => {
+const IFramePermissionKeys = set(['downloads', 'forms', 'modals', 'popups', 'fullscreen', 'camera', 'microphone', 'geolocation', 'payment', 'topNavigation', 'sameOrigin'])
```

Comment:
does this need to be `new Set()`? 

I get a set is not defined error

Reply:
Oops brainfart - yes!